src/transformers/models/detr/convert_detr_to_pytorch.py

```diff
@@ -344,9 +348,6 @@ def convert_detr_checkpoint(model_name, pytorch_dump_folder_path):
     original_outputs = detr(pixel_values)
     outputs = model(pixel_values)
 
-    print("Logits:", outputs.logits[0, :3, :3])
-    print("Original logits:", original_outputs["pred_logits"][0, :3, :3])
```

Comment:
Why remove this?

Reply:
I usually avoid these print statements on main